internal/cli/atlas/backup/restores_list.go

```diff
@@ -66,8 +67,11 @@ func RestoresListBuilder() *cobra.Command {
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
 			"args":            "clusterName",
+			"requiredArgs":    "clusterName",
 			"clusterNameDesc": "Name of the Atlas cluster for which you want to retrieve restore jobs.",
 		},
+		Example: fmt.Sprintf(`The following example retrieves the continuous backup restore jobs for the cluster Cluster0:
+  $ %s backup restore list Cluster0`, cli.ExampleAtlasEntryPoint()),
```

Comment:
Could you update the examples in `atlas cluster create`, `atlas cluster update`, `dbuser create`, `accesslist create ` to use `cli.ExampleAtlasEntryPoint()`

Reply:
> Could you update the examples in atlas cluster create, atlas cluster update, dbuser create, accesslist create to use cli.ExampleAtlasEntryPoint()?

Can we do that as a separate PR?

Reply:
sure